documentation/modules/proc-installing-using-olm.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// assembly-installing.adoc
+
+[id='installing-using-olm-{context}']
+= Installing {ProductName} using Operator Marketplace
+
+If the version of {ProductName} you want to install is not available via the link:https://www.operatorhub.io[Operator
+Hub], you can install a custom `OperatorSource` for {ProductName} following these steps.
+
+.Prerequisites
+
+* Operator Lifecycle Manager must be installed.
```

Comment:
Should we include links for more information on Operator Lifecycle Manager and Operator Marketplace? (Assuming an external site exists that we could point to)

Reply:
Added